packages/api/test/__fixtures__/sdk/simple-js-esm/schemas.d.ts

```diff
@@ -0,0 +1,292 @@
+declare const ApiResponse: {
+  readonly type: 'object';
```

Comment:
I hate this `readonly` stuff but this is what TS does when you generate a declaration file for an exported constant.